src/Input.test.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { fireEvent, render } from '@testing-library/react';
+
+import Input from './Input';
+
+describe('Input', () => {
+  const handleClick = jest.fn();
+  const handleChange = jest.fn();
+  const value = 'test';
+  const InputComponent = () => render(
+    <Input
+      value={value}
+      onClick={handleClick}
+      onChange={handleChange}
+    />,
+  );
```

Comment:
```suggestion
  const handleClick = jest.fn();
  const handleChange = jest.fn();

  const value = 'test';
  
  const InputComponent = () => render((
    <Input
      value={value}
      onClick={handleClick}
      onChange={handleChange}
    />
  ));
```

의미상으로 구분되는 것들은 공백을 넣어서 코드가독성을 높여주시면 좋아요.